docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,5 +1,27 @@
 # Parameters
 
-!!! warning
+## ROS parameters
 
-    Under Construction
+The ROS packages in Autoware may have ROS parameter(s) for setting some variables that should be customized depending on the applications.
+It is recommended not to set default values when you declare ROS parameters to avoid unintended behaviors.
+
+## ROS Parameter files
+
+Autoware has the following two types of parameter files for ROS packages:
+
+- Node parameter file: the reference parameter for the node package
+  - e.g., [the parameter for the `behavior_path_planner` package](https://github.com/autowarefoundation/autoware.universe/tree/main/planning/behavior_path_planner/config)
+- Launch parameter file: [the reference parameter for the `autoware_launch` package](https://github.com/autowarefoundation/autoware_launch/tree/main/autoware_launch/config)
+
+All the parameter files should have the `.param.yaml` suffix so that the auto-format can be applied properly.
+
+### Node parameter
+
+Node parameter files store the default parameters provided for each package in Autoware Universe.
+All the nodes in Autoware **must** have the node parameter file.
```

Comment:
[Question] For what reason?

Reply:
I thought it is mandatory so that all the nodes should be able to be tuned depending on the vehicles. But then I realized this is not always the case (some node may have no parameters, and we don't need parameter files in such case).

Is this also what your question intended? Then, I will fix this line.

Reply:
Yes. I wanted to know the background reason.
Let's write the rationales and exceptions.

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/271/commits/7187ce7bd0d4439a716fb1842ddd476dfb7cc25e
How about this?

Reply:
Cool!

[nits]
- no comma?
- the node have -> the node has?

Reply:
As well as resolving the grammar issue, I also revised the sentence https://github.com/autowarefoundation/autoware-documentation/pull/271/commits/f420a284eca97bd8a5d203cb85d9af67859d1d8f

Reply:
:+1: